### ISSUE TITLE
Method 'insert' cannot be called on 'OverlayState?' because it is potentially null

### ISSUE THREAD

**Issue by obrunsmann** [Mayb3Nots/zoom_pinch_overlay#9]
Installed version: "1.3.1+1"

```
: Error: Method 'insert' cannot be called on 'OverlayState?' because it is potentially null.
../…/lib/zoom_pinch_overlay.dart:232
- 'OverlayState' is from 'package:flutter/src/widgets/overlay.dart' ('../../../flutter/packages/flutter/lib/src/widgets/overlay.dart').
package:flutter/…/widgets/overlay.dart:1
    Try calling using ?. instead.
          overlayState.insert(_overlayEntry!);
                       ^^^^^^
```

**Doctor**
```
[✓] Flutter (Channel stable, 3.3.4, on macOS 12.6 21G115 darwin-arm, locale de-DE)
    • Flutter version 3.3.4 on channel stable at /Users/oliver/flutter
    • Upstream repository https://github.com/flutter/flutter.git
    • Framework revision eb6d86ee27 (5 days ago), 2022-10-04 22:31:45 -0700
    • Engine revision c08d7d5efc
    • Dart version 2.18.2
    • DevTools version 2.15.0

[!] Android toolchain - develop for Android devices (Android SDK version 32.1.0-rc1)
    • Android SDK at /Users/oliver/Library/Android/sdk
    ✗ cmdline-tools component is missing
      Run `path/to/sdkmanager --install "cmdline-tools;latest"`
      See https://developer.android.com/studio/command-line for more details.
    ✗ Android license status unknown.
      Run `flutter doctor --android-licenses` to accept the SDK licenses.
      See https://flutter.dev/docs/get-started/install/macos#android-setup for more details.

[✓] Xcode - develop for iOS and macOS (Xcode 14.0.1)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Build 14A400
    • CocoaPods version 1.11.3

[✓] Chrome - develop for the web
    • Chrome at /Applications/Google Chrome.app/Contents/MacOS/Google Chrome

[✓] Android Studio (version 2021.2)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build 11.0.12+0-b1504.28-7817840)

[✓] VS Code (version 1.72.0)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension version 3.50.0

[✓] Connected device (3 available)
xxx

[✓] HTTP Host Availability
    • All required HTTP hosts are available

```

**Comment by obrunsmann** [comment]
Reverted to zoom_pinch_overlay: 1.1.2 where build succeeds btw

**Comment by Mayb3Nots** [comment]
What version of dart and flutter are you using?

**Comment by Mayb3Nots** [comment]
Thanks for opening this issue. My flutter version was on master that's why.
Closed via 679167747be243878ff2637cd18e87c19017f119
